### DISCUSSION TITLE
Mulesoft: Messages were not delivering from exchange to queue using json logger Destination

### DISCUSSION THREAD

**Discussion by hema4421** [rabbitmq/rabbitmq-server#12844] [Questions] [answered]
### Describe the bug

my messages were not sending to rabbitmq queues through exchange when i am sending through json logger destination in Mule.
tried with exchange type fanout and direct ?

### Reproduction steps

1.install rabbit mq in local system, created simple queue(test1) and exchange in rabbit mq test them locally
2.create a mule project with amqp listener configured with queue then add your json logger available in your organisation 
3.configure json logger destination with amqp details 
4.send msg to test1 queue from rabbitmq our mule should trigger and reflect back the msgs to exchange bind queues
5.here the the msgs were not reflect back from exchange to queue is the issue from json logger

...
![error](https://github.com/user-attachments/assets/17fc31df-2655-41e1-b235-09aefd22e3c7)

### Expected behavior

here the the msgs were not reflect back from exchange to queue is the issue from json logger

**Comment by mkuratczyk** [comment, marked as answer]
This is a Mule question/issue, unless you can reproduce it without Mule.
